evennia/web/webclient/static/webclient/js/default_io.js

```diff
@@ -0,0 +1,41 @@
+/*
+ *
+ * Evennia Webclient default 'send-text-on-enter-key' IO plugin
+ *
+ */
+plugin_handler.add('defaultio', (function () {
```

Comment:
Style peeve that is probably more of a personal thing than anything: I know JS code loves to do these nested function calls, but they lead to this bunch of parentheses and curly brackets at the bottom which is just hard to parse and check. Would it not be more readable to define the function in a variable first and then do the `plugin_handler.add()` call on its own near the bottom? Not necessary to change, but maybe worth considering?